src/OrchardCore/OrchardCore.ResourceManagement.Core/ResourceSetting.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using OrchardCore.Settings;
+
+namespace OrchardCore.ResourceManagement.Core;
+
+public class ResourceSetting
```

Comment:
This class should have the `Options` suffix.

Reply:
Makes sense. Renamed :)